mysite/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+USER_CHOICES = [
+    ('freelancer', "I'm a Freelancer"),
+    ('recruiter', "I'm a Recruiter"),
+    ]
+
+
+class User(AbstractUser):
+    user_role = models.CharField(max_length=100, choices=USER_CHOICES, default='freelancer')
+
+
+class Project(models.Model):
+    created_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    name = models.CharField(max_length=30)
+    description = models.CharField(max_length=100)
+    status = models.CharField(max_length=30)
+    assigned_to = models.CharField(max_length=30)
```

Comment:
what type of value assigned_to is holding. If its a user id, it should be ForeignKey. I think you should also put budget filed in this model.

Reply:
assigned_to is holding username. Its not holding user_id, that's why it is not ForeignKey.